packages/app/app/layout.tsx

```diff
@@ -0,0 +1,47 @@
+// These styles apply to every route in the application
+import "../styles/global.css";
+
+export const metadata = {
+  title: "Stackly UI",
+  description:
+    "Stackly is a Next.js Cloudflare mono repo project that implements Dollar Cost Averaging using CoW protocol.",
+};
+
+interface RootLayoutProps {
+  children: React.ReactNode;
+}
+
+export default function RootLayout({ children }: RootLayoutProps) {
+  return (
+    <html lang="en">
+      <head>
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href="/assets/favicons/apple-touch-icon.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="32x32"
+          href="/assets/favicons/favicon-32x32.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="16x16"
+          href="/assets/favicons/favicon-16x16.png"
+        />
+        <link rel="manifest" href="/assets/favicons/site.webmanifest" />
+        <link
+          rel="mask-icon"
+          href="/assets/favicons/safari-pinned-tab.svg"
+          color="#F8F8F7"
+        />
+        <meta name="msapplication-TileColor" content="#F8F8F7" />
+        <meta name="theme-color" content="#A2E771" />
+      </head>
```

Comment:
with new app router we don't need that to set up the icons.

Just add the favicon.ico to the root of the project

Reply:
That's why I added the favicon.icon to the public but you need to code the below files for apple and icon specific to devices based on the size for device specific responses.

`Add an icon.(js|ts|tsx) or apple-icon.(js|ts|tsx) file that default exports a function that returns Blob | ArrayBuffer | TypedArray | DataView | ReadableStream | Response.`

 Or use what I did above. I thought the above was easy

Reply:
That is what I understand from the doc. If I am wrong please advice. I can change.

Reply:
You can put the favicon.icon on the root folder (not in public) and it will work.

Reply:
With app router you just need to add the correct filenames to work:

Just add this files to the root:
- favicon.ico
- icon.(jpg|jpeg|png|svg)
- apple-icon.(jpg|jpeg|png|svg)

And you can remove this `head`.

Docs:
<img width="672" alt="image" src="https://github.com/ImpeccableHQ/stackly-ui/assets/5664434/d40cfff8-5e8e-47d5-ab41-0328b0f21977">